contrib/hbmxml/tests/tohash.prg

```diff
@@ -53,8 +53,13 @@ STATIC FUNCTION s_NodeToHash( pNode )
                   AAdd( hHash[ mxmlGetElement( pNode ) ], hHashChild )
                ENDIF
             ELSEIF hb_mxmlGetAttrsCount( pNode ) > 0
-               hHash[ mxmlGetElement( pNode ) + "@attr" ] := hb_mxmlGetAttrs( pNode )
+                  IF Empty( hHash[ mxmlGetElement( pNode ) ] ) 
+                        hHash[ mxmlGetElement( pNode ) ] := {}
```

Comment:
No, it's fine here, but since `hbformat` tends to be destructive with manually (well-) formatted sources, I strongly suggest to correct these few lines manually.